src/katsdpcontroller/schemas/product_config.json.j2

```diff
@@ -611,6 +611,13 @@
             "additionalProperties": false,
             "properties": {
                 "develop": {"type": "boolean", "default": false},
+{% if version >= "3.2" %}
```

Comment:
To make a version 3.2, you need to add it to the list of accepted versions at the top of the file.

Reply:
Done